code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/impl/GithubOAuthImpl.java

```diff
@@ -41,7 +42,7 @@ public Account getAccount(ApiRequest request) {
             return null;
         }
         String token = request.getServletContext().getRequest().getHeader(AUTH_HEADER);
-        if (StringUtils.isEmpty(token)) {
+        if (StringUtils.isEmpty(token) || !token.startsWith(AUTH_TYPE)) {
```

Comment:
Pedantic: This should be a case-insensitive test, and check for/split on whitespace after the AUTH_TYPE.
`Authorization: bearer jwtdata` should match
`Authorization: bearerfoobar somethingelse` shouldn't